app/client/src/utils/utils.js

```diff
@@ -72,6 +72,8 @@ function titleCaseWithExceptions(string: string) {
   switch (string) {
     case 'AMMONIA, UN-IONIZED':
       return 'Ammonia, Un-Ionized';
+    case 'BIOCHEMICAL OXYGEN DEMAND (BOD)':
+      return 'Biochemical Oxygen Demand (BOD)';
```

Comment:
You don't need to worry about these. We can leave this one in, but EPA just tells us the ones they would like to be mapped through the `titleCaseWithExceptions` function. EPA just wanted us to only map the high priority ones, since this would be a large list that could change fairly often.